### PR TITLE
OMERO 5.4.3 release

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/AisCSEBR/41-open-pr-against-wwwopenmicroscopyorg

Current implementation means people oblivious to the 5.4.2 thing will need an extra click to see the actual updates they are getting from 5.4.1 - not sure if we want to reproduce the 5.4.2 text or do some slightly shifty re-gigging of older post to combine 5.4.2 and 5.4.3 announcements - this seemed the most transparent way to do it but is more effort for some readers.

### CODE REVIEW

**Comment by hflynn** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/omero/downloads/ and https://snoopycrimecop.github.io/www.openmicroscopy.org/2018/01/30/omero-5-4-3.html

File: _posts/2018-01-30-omero-5-4-3.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: Release of OMERO 5.4.3
+intro-blurb: The OME team announces the release of OMERO 5.4.3.
+---
+Today we are releasing OMERO 5.4.3.
+
+This release is the same as [5.4.2]({{ site.baseurl }}/2018/01/23/omero-5-4-2.html) but with a fix in place for the OMERO.web
+issue.
```
Comment (mtbc) [inline _posts/2018-01-30-omero-5-4-3.md]: Perhaps link "the ... issue" to https://snoopycrimecop.github.io/www.openmicroscopy.org/2018/01/26/omero-5-4-2-issue.html ?

**Comment by hflynn** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/2018/01/30/omero-5-4-3.html updated

**Comment by hflynn** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/announcements/ updated with today's date now

**Comment by rleigh-codelibre** [comment]
Looks fine to me.

**Pull request merged**